packages/vscode-extension/src/common/Project.ts

```diff
@@ -215,6 +218,8 @@ export interface ProjectInterface {
   renameDevice(deviceInfo: DeviceInfo, newDisplayName: string): Promise<void>;
   updatePreviewZoomLevel(zoom: ZoomLevelType): Promise<void>;
 
+  setLaunchConfiguration(launchConfig: LaunchConfigurationOptions): Promise<void>;
```

Comment:
maybe `selectLaunchConfigurationWithOptions` ? We don't actually send launch config object but options here

Reply:
`Options` is just `LaunchConfiguration` that you don't have to specify the optional fields for. I think it's fine. I changed the name to `select`.